src/ml_gym/frontend/dashboard/src/routes/analysis_board.tsx

```diff
@@ -0,0 +1,126 @@
+import "./analysis_board.css";
+
+import React from "react";
+import {
+    LineChart,
+    Line,
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
  Line,
```